### PR TITLE
Add Kubernetes 1.11 release manager interview

### PR DESCRIPTION
**Pull request by craigbox**
This blog post is a transcript of the interview with Kubernetes 1.11 release manager Josh Berkus and 1.12 release manager Tim Pepper, [recently released on the Kubernetes Podcast](https://kubernetespodcast.com/episode/010-kubernetes-1.11).

### CODE REVIEW

File: content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+layout: blog
+title: "How the sausage is made: the Kubernetes 1.11 release interview, from the Kubernetes Podcast"
+date:  Monday, July 16, 2018
+---
+
+<b>Author</b>: Craig Box (Google)
+
+At KubeCon EU, my colleague Adam Glick and I were pleased to announce the [Kubernetes Podcast from Google](https://kubernetespodcast.com/). In this weekly conversation, we focus on all the great things that are happening in the world of Kubernetes and Cloud Native. From the news of the week, to interviews with people in the community, we help you stay up to date on everything Kubernetes.
+
+We [recently had the pleasure of speaking](https://kubernetespodcast.com/episode/010-kubernetes-1.11/) to the release manager for Kubernetes 1.11, Josh Berkus from Red Hat, and his deputy, the release manager for the upcoming 1.12, Tim Pepper from VMware.
```
Comment (neolit123) [inline content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md]: do we really need to add `his deputy` here? :)
>,and the release manager for the upcoming 1.12,

there is definitely a release lead for a release, but i doubt they call anyone else on the release team a deputy.
the is the notion of a "shadow", but Tim wasn't shadowing Josh for 1.11.

File: content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+
+<b>CRAIG BOX: Speaking of cultural changes, professional baker to Kubernetes' release lead sounds like quite a journey.</b>
+
+JOSH BERKUS: There was a lot of stuff in between. 
+
+<b>CRAIG BOX: Would you say there are a lot of similarities?</b>
+
+JOSH BERKUS: You know, believe it or not, there actually are similarities. And here's where it's similar, because I was actually thinking about this earlier. So when I was a professional baker, one of the things that I had to do was morning pastry. Like, I was actually in charge of doing several other things for custom orders, but since I had to come to work at 3:00 AM anyway— which also distressingly has similarities with some of this process. Because I had to come to work at 3:00 AM anyway, one of my secondary responsibilities was traying the morning pastry.
```
Comment (neolit123) [inline content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md]: `traying` -> `trying`.

File: content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md (hunk 1 above)
Comment (tpepper) [reply content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md]: Formally Lubomir is correct as I did not shadow Josh's lead.

File: content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+
+TIM PEPPER: So I guess it's a term in the art, but for those who aren't familiar with it, you have intended behavior that wasn't covered by test, and as a result, an unintended change happens to that. And instead of your intended behavior being shipped, you're shipping something else. 
+
+JOSH BERKUS: The JSON change is a textbook example of this, which is we were testing that the API would continue to accept correct JSON. We were not testing adequately that it wouldn't except incorrect JSON. 
+
+TIM PEPPER: A test escape, another way to think of it as you shipped a bug because there was not a test case highlighting the possibility of the bug. 
+
+<b>ADAM GLICK: It's the classic, we tested to make sure the feature worked. We didn't test to make sure that breaking things didn't work.</b>
+
+TIM PEPPER: It's common for us to focus on, I've created this feature and I'm testing the positive cases. And this also comes to thinking about things like secure by default and having a really robust system that a harder piece of engineering often is to think about the failure cases and really actively manage those well.
```
Comment (tpepper) [inline content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md]: There's a missing punctuation between "...having a really robust system" and "...That a harder piece of engineering".

File: content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+
+JOSH BERKUS: Well, we just went through this whole retro, and I put in some recommendations for things. Obviously, some additional automation, which I'm going to be looking at doing now that I'm cycling off of the release team for a quarter and can actually look at more longer term goals, will help, particularly now that we've addressed actually some of our GitHub data flow issues. 
+
+Beyond that, I put in a whole bunch of recommendations in the retro, but it's actually up to Tim which recommendations he's going to try to implement. So I'll let him [comment]. 
+
+TIM PEPPER: I think one of the biggest changes that happened in the 1.11 cycle is this emphasis on trying to keep our continuous integration test status always green. That is huge for software development and keeping velocity. If you have this more— I guess at this point antiquated notion of waterfall development where you do feature development for a while and are accepting of destabilization, and somehow later you're going to come back and spend a period on stabilization and fixing, that really elongates the feedback loop for developers.
```
Comment (tpepper) [inline content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md]: Suggest different punctuation, eg:

"If you have this more, I guess at this point antiquated notion of waterfall development, where you..."

File: content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+
+Beyond that, I put in a whole bunch of recommendations in the retro, but it's actually up to Tim which recommendations he's going to try to implement. So I'll let him [comment]. 
+
+TIM PEPPER: I think one of the biggest changes that happened in the 1.11 cycle is this emphasis on trying to keep our continuous integration test status always green. That is huge for software development and keeping velocity. If you have this more— I guess at this point antiquated notion of waterfall development where you do feature development for a while and are accepting of destabilization, and somehow later you're going to come back and spend a period on stabilization and fixing, that really elongates the feedback loop for developers. 
+
+And they don't realize what was broken, and the problems become much more complex to sort out as time goes by. One, developers aren't thinking about what it was that they'd been working on anymore. They've lost the context to be able to efficiently solve the problem.
```
Comment (tpepper) [inline content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md]: "the problems become" -> "the problem's become"

File: content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md [hunk 6]
```diff
@@ -0,0 +1,275 @@
...
+
+JOSH BERKUS: OK. OK. Maybe not then. 
+
+[LAUGH] 
+
+TIM PEPPER: It's potentially becoming a lot of cats to herd. But I think even outside of that formal volunteering to be a named shadow, anybody is welcome to show up to the release team meetings, follow the release team activities on [Slack](http://slack.k8s.io), start understanding how the process works. And really, this is the case all across open source. It doesn't even have to be the release team. If you're passionate about networking, start following what SIG Network is doing. It's the same sort of path, I think, into any area on the project. 
+
+Each of the SIGs [has] a channel. So it would be #SIG-whatever the name is. [In our] case, #SIG-Release. 
+
+I'd also maybe give a plug for a [talk I did at KubeCon](https://github.com/kubernetes/community/tree/master/sig-scalability) in Copenhagen this spring, talking about how the release team specifically can be a path for new contributors coming in. And had some ideas and suggestions there for newcomers.
```
Comment (tpepper) [inline content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md]: Slides link: https://schd.ws/hosted_files/kccnceu18/3e/KubeConEU2018%20Growing%20in%20Your%20Contributor%20Role.pdf

Video link:
https://youtu.be/goAph8A20gQ

File: content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md [hunk 7]
```diff
@@ -0,0 +1,275 @@
...
+
+First of all, what went well?</b>
+
+JOSH BERKUS: Two things, I think, really improved things, both for contributors and for the release team. Thing number one was putting a strong emphasis on getting the test grid green well ahead of code freeze. 
+
+TIM PEPPER: Definitely. 
+
+JOSH BERKUS: Now partly that went well because we had a spectacular CI lead, [Aish Sundar](https://github.com/aishsundar), who's now in training to become the release lead. 
+
+TIM PEPPER: And I'd count that partly as one of the, where were you lucky? We happened upon a wonderful person who just popped up and volunteered.
```
Comment (tpepper) [inline content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md]: ...one of the "Where were you lucky?" areas.

File: content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md [hunk 8]
```diff
@@ -0,0 +1,275 @@
...
+
+
+<hr/>
+
+<i>Thanks to Josh Berkus and Tim Pepper for talking to the Kubernetes Podcast from Google.
+
+[Josh Berkus](https://github.com/jberkus) hangs out in #sig-release on the [Kubernetes Slack](slack.k8s.io). He maintains a newsletter called "[Last Week in Kubernetes Development](http://lwkd.info/)", with Noah Kantrowitz. You can read him on Twitter at [@fuzzychef](https://twitter.com/fuzzychef), but he does warn you that there's a lot of politics there as well.
+
+
+[Tim Pepper](https://github.com/tpepper) is also on Slack - he's always open to folks reaching out with a question, looking for help or advice.  On Twitter you'll find him at [@pythomit](https://twitter.com/pythomit), which is "Timothy P" backwards. Tim is an avid soccer fan and season ticket holder for the [Portland Timbers](https://www.timbers.com/) and the [Portland Thorns](https://www.timbers.com/thornsfc), so you'll get all sorts of opinions on soccer in addition to technology!
```
Comment (tpepper) [inline content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md]: https://portlandtimbers.com and https://portlandthorns.com are probably more stable resolving links across redirects and hosting changes.

File: content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md (hunk 2 above)
Comment (craigbox) [reply content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md]: Traying is correct; "putting pastries on a tray".

File: content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md (hunk 3 above)
Comment (craigbox) [reply content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md]: Fixed

File: content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md [hunk 9]
```diff
@@ -150,9 +150,9 @@ JOSH BERKUS: Well, we just went through this whole retro, and I put in some reco
...
+TIM PEPPER: I think one of the biggest changes that happened in the 1.11 cycle is this emphasis on trying to keep our continuous integration test status always green. That is huge for software development and keeping velocity. If you have this more, I guess at this point antiquated notion of waterfall development, where you do feature development for a while and are accepting of destabilization, and somehow later you're going to come back and spend a period on stabilization and fixing, that really elongates the feedback loop for developers. 
 
-And they don't realize what was broken, and the problems become much more complex to sort out as time goes by. One, developers aren't thinking about what it was that they'd been working on anymore. They've lost the context to be able to efficiently solve the problem. 
+And they don't realize what was broken, and the problem's become much more complex to sort out as time goes by. One, developers aren't thinking about what it was that they'd been working on anymore. They've lost the context to be able to efficiently solve the problem.
```
Comment (tpepper) [inline content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md]: shouldn't be possessive here

**Comment by craigbox** [comment]
@kbarnard10 I had hoped to publish this on Monday while it's timely.  I think Kristen has mentioned it to you.  Is that OK by you?

File: content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md [hunk 10]
```diff
@@ -0,0 +1,275 @@
+---
+layout: blog
+title: "How the sausage is made: the Kubernetes 1.11 release interview, from the Kubernetes Podcast"
+date:  Monday, July 16, 2018
```
Comment (kbarnard10) [inline content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md]: Can you change date format to: 2018-07-16

Comment (craigbox) [reply content/en/blog/_posts/2018-07-16-kubernetes-1.11-release-interview.md]: Done

**Pull request merged**